fileinfo.go

```diff
@@ -43,6 +67,25 @@ func SetFileBasicInfo(f *os.File, bi *FileBasicInfo) error {
 	return nil
 }
 
+// FileStandardInfo contains extended information for the file.
+// FILE_STANDARD_INFO in WinBase.h
+// https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-fscc/5afa7f66-619c-48f3-955f-68c4ece704ae
+type FileStandardInfo struct {
+	AllocationSize, EndOfFile int64
+	NumberOfLinks             int32
```

Comment:
Looks like this should be `uint32`.

https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/wdm/ns-wdm-_file_standard_information

Reply:
Ah, so it is. It even says that in the link directly above the structure, I think I flipped it by accident when I flipped the previous two, which I originally recall writing as `uint64`.